tests/tracer/test_encoders.py

```diff
@@ -259,6 +259,34 @@ def test_encode_traces_msgpack_v03(self):
                 assert b"client.testing" == items[i][j][b"name"]
 
 
+@pytest.mark.parametrize("version", ["v0.3", "v0.4"])
+def test_encode_meta_struct(version):
+    # test encoding for MsgPack format
+    encoder = MSGPACK_ENCODERS[version](2 << 10, 2 << 10)
+    super_span = Span(name="client.testing", trace_id=1)
+    payload = {"tttt": {"iuopç": [{"abcd": 1, "bcde": True}, {}]}, "zzzz": b"\x93\x01\x02\x03", "ZZZZ": [1, 2, 3]}
+
+    super_span.set_struct_tag("payload", payload)
+    super_span.set_tag("payload", "meta_payload")
+    encoder.put(
+        [
+            super_span,
+            Span(name="client.testing", trace_id=1),
+        ]
+    )
+
+    spans = encoder.encode()
+    print(spans)
```

Comment:
Removing this would make test output in CI a little more readable.

Reply:
Of course, it's removed.